README.md

```diff
@@ -36,6 +36,60 @@ Incidents are typically categorized among following types:
 -   **Infrastructure Event**
     -   Used for integration for system management tools.
 
+The **Authentication Mechanism to Use** config parameter provides the configuration information Phantom uses to communicate with BMC Platform. Currently, there are two ways to authenticate.
+
+-   Basic
+-   BMC SSO (Interactive)
+
+ 
+## Basic Authentication
+
+Fill in the **Base URL** , **Username** and **Password**. The remaining configuration
```

Comment:
The remaining configuration - feels like this sentence has ended mid way

Reply:
Will update it.